bloom-housing/bloom
# fix: updates around cookies

# Pull Request Template

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/2250)


## Description
This addresses some issues that came up in bug bash mainly: 
1. locally code is not used throughout instead of a mix of code and token around the mfa code that is sent out. I think a db query that updates this on the hba env needs to be built out

2. a small fix around the hooks that swr was using. This should reduce the fail rate of the hooks in general, but is not a fix for the useSWR issues we were seeing 

3. added a site alert so the account confirmation alert can be consumed properly after log in

## How Can This Be Tested/Reviewed?
create a new user
log in as that user
notice on the page immediately after accepting the terms lets you know that the account was confirmed via a site alert

when you request an mfa code your email should call it a code instead of a token

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
